docs/pages/machine-id/getting-started.mdx

```diff
@@ -93,10 +93,14 @@ editor                                                            user:list,crea
 
 Machine ID can join with a token or the [IAM Method](https://goteleport.com/docs/setup/guides/joining-nodes-aws) on AWS.
 
+Assuming that you are using the default `access` role, ensure that you use the 
+`--logins` flag when adding your bot to specify the SSH logins that you wish to 
+allow the bot to access on hosts. For our example, we will be using `root`.
+
 <Tabs>
   <TabItem label="Token-based Joining">
   ```code
-  $ tctl bots add robot --roles=access
+  $ tctl bots add robot --roles=access --logins=root
```

Comment:
I'm not sure if this would flow well in our existing documentation, but do we want to mention the side-effects of using logins from trait vs role anywhere? Configuration reference maybe?

Reply:
Yeah, or perhaps in FAQ.

Reply:
I'm going to raise that as a seperate PR 👍